### ISSUE TITLE
Consolidate scientist image file locations

### ISSUE THREAD

**Issue by becky-gilbert** [lookit/lookit-api#1102]
**TL;DR**: The images on The Scientists page come from various locations (cloud and self-hosted). We should consider simplifying/consolidating the image locations.

**Narrative**

The images on [The Scientists page](https://lookit.mit.edu/scientists/) are loading from a variety of places: AWS S3 lookit contents, Square Space CDN, Lookit-API static folder, Google API lookit-staging.

As a developer, it would be easier to find/edit these image files and decide where to put new ones if they were all stored in the same place. We've acknowledged this issue for self-hosted files (#852) and it makes sense to look at where we are storing externally-hosted content as well. It's not clear to me whether this is just the case for The Scientists page, or if there are similar issues in other places throughout the site. As a bonus, we might be able to eliminate one or more cloud storage sources from our project entirely.

**Acceptance Criteria**

- [ ] Scoping: should we just address the Scientist image files, or go through all external links to see if this can be improved elsewhere throughout the site?
- [ ] Consider alternatives: do nothing :)
- [ ] Decide where to put image files and move them there.
- [ ] Test that links work and images load (locally, and on staging/production).

**Implementation Notes**

Some things to consider: storage costs, ease of adding/updating files, possibility of eliminating one or more storage sources entirely.

This issue applies to content under our own control, but NOT versioned files that are managed by others - those we should expect to get from many different sources (CDNs) and is not a problem.
